### PR TITLE
check api key valid

### PR DESCRIPTION
**Pull request by tejaswinichile**
Issue #427

### CODE REVIEW

**Comment by tejaswinichile** [comment]
@breezyboa _a Please check

File: app/javascript/Client/Main.elm [hunk 1]
```diff
@@ -93,8 +99,10 @@ init flags location =
       , searchQuery = ""
       , history = NoHistory
       , userInfo = { name = "", email = "" }
+      , renderHelpButton = True
```
Comment (breezyboa) [inline app/javascript/Client/Main.elm]: initial value should be False. Only after confirmation should this be true

File: app/javascript/Client/Main.elm [hunk 2]
```diff
@@ -424,19 +439,24 @@ onTabChange tab model =
...
+                    , ( "color", "#fff" )
+                    ]
+                , onClick (SetAppState Maximized)
+                ]
+                [ div [ class "question-icon" ]
+                    [ SolidIcon.question ]
+                ]
+
+        _ ->
+            div [] []
```
Comment (breezyboa) [inline app/javascript/Client/Main.elm]: `text ""` instead of div

File: app/javascript/Client/Data/Organization.elm [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+organizationQuery : GQLBuilder.Document GQLBuilder.Query Organization { vars | apiKey : String }
+organizationQuery =
+    let
+        apiKeyVar =
+            Var.required "apiKey" .apiKey Var.string
+    in
+        GQLBuilder.queryDocument <|
+            GQLBuilder.extract
+                (GQLBuilder.field "organizations"
+                    [ ( "api_key", Arg.variable apiKeyVar ) ]
```
Comment (breezyboa) [inline app/javascript/Client/Data/Organization.elm]: is this not redundant? api key is anyway present in the header

**Comment by tejaswinichile** [comment]
@breezyboa _a Please check.

**Pull request merged**